.github/workflows/ci-xcode.yml

```diff
@@ -32,7 +32,7 @@ jobs:
     runs-on: macos-13
     strategy:
       matrix:
-        xcode: ["14.3.1"]
+        xcode: ["14.3.1", "15.2"]
```

Comment:
Github actions doesn't yet support 15.2 - https://github.com/actions/runner-images/pull/9169

We'll see this test failing until that's merged.

Reply:
But it is mentioned [here](https://github.com/actions/runner-images/blob/releases/macos-13/20240114/images/macos/macos-13-Readme.md) 🤔

Reply:
Oh. Well, excellent. Guess the thread I found was already out of date.

Reply:
Well, that's annoying. Is the Vision Pro simulator not created by default? https://github.com/Quick/Nimble/actions/runs/7554289300/job/20585517470?pr=1098

Reply:
It looks like it 😞 

[6eb5881](https://github.com/Quick/Nimble/pull/1098/commits/6eb588189a4941efe9eaab83e60ba78aa5ee1c9e) I reverted it to unblock this PR. Will keep an eye on the GH runner images and add it once it changes.

Reply:
Just found [this issue](https://github.com/actions/runner-images/issues/9139) where they point out that the version of Xcode 15.2 is the beta 2 and not the final version. I guess it should be fine once they use the stable version of 15.2.

Reply:
I'll watch that thread and update our CI once a new build is out.